HOW_WE_USE_GITHUB.md

```diff
@@ -91,83 +95,110 @@ At the most basic "bird's eye view" level, sorted issues will fall into the cate
+### What is the purpose of having a "Backlog"?
 
-Once issues are deemed ready to be worked on, they will be moved to the [`conda` Backlog tab of the Planning board][project-backlog] on GitHub. Once actively in progress, the issues will be moved to the [In Progress tab of the Planning board][project-in-progress] and then closed out once the work is complete.
+Issues are "backlogged" when they have been sorted but not yet earmarked for an upcoming release.
 
+### What automation procedures are currently in place?
 
-#### What is the purpose of having a "Backlog"?
+Global automation procedures synced out from the [`conda/infrastructure`][infrastructure] repo include:
 
-Issues are "backlogged" when they have been sorted but not yet earmarked for an upcoming release. Weekly Refinement meetings are a time when the `conda` engineers will transition issues from "[Sorting][project-sorting]" to "[Backlog][project-backlog]". Additionally, this time of handoff will include discussions around the kind of issues that were raised, which provides an opportunity to identify any patterns that may point to a larger problem.
+- [Marking of issues and pull requests as stale][workflow-stale], resulting in:
+  - issues marked as https://github.com/conda/infrastructure/labels/type%3A%3Asupport being labeled stale after 21 days of inactivity and being closed after 7 further days of inactivity (that is, closed after 30 inactive days total)
+  - all other inactive issues (not labeled as https://github.com/conda/infrastructure/labels/type%3A%3Asupport) being labeled stale after 365 days of inactivity and being closed after 30 further days of inactivity (that is, closed after an approximate total of 1 year and 1 month of inactivity)
+  - all inactive pull requests being labelled stale after 365 days of inactivity and being closed after 30 further days of inactivity (that is, closed after an approximate total of 1 year and 1 month of inactivity)
+- [Locking of closed issues and pull requests with no further activity][workflow-lock] after 365 days
+- [Adding new issues and pull requests to the respective project boards][workflow-project]
+- [Indicate an issue ready for the sorting engineer's attention by toggling https://github.com/conda/infrastructure/labels/pending%3A%3Afeedback with https://github.com/conda/infrastructure/labels/pending%3A%3Asupport after a contributor leaves a comment][workflow-issues]
+- [Verifying that contributors have signed the CLA][workflow-cla] before allowing pull requests to be merged
+- [Syncing out templates, labels, workflows and documentation][workflow-sync] from [`conda/infrastructure`][infrastructure] to the other repositories
```

Comment:
I would love a serial comma here before the "and".

Reply:
```suggestion
- [Syncing out templates, labels, workflows, and documentation][workflow-sync] from [`conda/infrastructure`][infrastructure] to the other repositories
```